package.json

```diff
@@ -87,7 +87,7 @@
     "core-js": "3.6.4",
     "gherkin": "9.0.0",
     "glob": "7.1.6",
-    "lodash": "4.17.19",
+    "lodash": "4.17.20",
```

Comment:
As this is a dev-time tool it may be better to loosen the dependency version lock a bit (e.g. to `^4.17.20`).

That way downstream module maintainers don't have to wait for this module to update minor versions of its dependencies and perform `npm upgrade lodash` for compatible updates on their own.

Reply:
The reason I lock dependencies is that it prevents gherkin-lint from breaking if one of the modules it depends on makes a bad new release. I see your point, but I've been bitten a lot in the past too, so I'm leaning towards keeping everything locked.

Fyi the prb failed because you didn't run `npm install` so the `package-lock.json` didn't update. I'll merge this and update it so I don't have to wait on you, then you have to wait on me again etc.

Reply:
@vsiakka Thank you!